src/views/administration/configuration/RiskScore.vue

```diff
@@ -0,0 +1,160 @@
+            unassigned: "5",
+            unassignedTooltip: "",
+          }
+        }
+      },
+      created () {
+        this.axios.get(this.configUrl).then((response) => {
+          let configItems = response.data.filter(function (item) { return item.groupName === "search-indexes" });
+          for (let i=0; i<configItems.length; i++) {
+            let item = configItems[i];
+            switch (item.propertyName) {
+              case "weight.history.enabled":
+                this.customRiskScore.enabled = common.toBoolean(item.propertyValue); break;
+              case "weight.critical":
+                this.customRiskScore.critical = item.propertyValue;
+                this.customRiskScore.criticalTooltip = item.description; break;
+                case "weight.high":
+                this.customRiskScore.high = item.propertyValue;
+                this.customRiskScore.highTooltip = item.description; break;
+                case "weight.medium":
+                this.customRiskScore.medium = item.propertyValue;
+                this.customRiskScore.mediumTooltip = item.description; break;
+                case "weight.low":
+                this.customRiskScore.low = item.propertyValue;
+                this.customRiskScore.lowTooltip = item.description; break;
+                case "weight.unassigned":
+                this.customRiskScore.unassigned = item.propertyValue;
+                this.customRiskScore.unassignedTooltip = item.description; break;
+            }
+          }
+        });
+      },
+      methods: {
+        saveCustomRiskScoreSettings: function() {
+          this.updateConfigProperties([
+            {groupName: 'risk-score', propertyName: 'weight.history.enabled', propertyValue: this.customRiskScore.enabled},
+            {groupName: 'risk-score', propertyName: 'weight.critical', propertyValue: this.customRiskScore.critical},
+            {groupName: 'risk-score', propertyName: 'weight.high', propertyValue: this.customRiskScore.high},
+            {groupName: 'risk-score', propertyName: 'weight.medium', propertyValue: this.customRiskScore.medium},
+            {groupName: 'risk-score', propertyName: 'weight.low', propertyValue: this.customRiskScore.low},
+            {groupName: 'risk-score', propertyName: 'weight.unassigned', propertyValue: this.customRiskScore.unassigned}
+          ]);
+        },
+        reindex: function() {
+          let url = `${this.$api.BASE_URL}/${this.$api.URL_SEARCH}/reindex`;
+          let params = new URLSearchParams();
+          Object.entries(this.type).forEach(([key, value]) => {
+            if(value) {
+              params.append('type', key.toUpperCase());
+            }
+          });
+          this.axios.post(url, null, { params: params }).then((response) => {
+            this.$toastr.s(this.$t('admin.reindex_submitted'));
+          }).catch((error) => {
+            this.$toastr.s(this.$t('admin.reindex_error'));
+          });
+        }
```

Comment:
I think this might be a copy-paste leftover? Let's remove this.

Reply:
removed